scripts/package/package_xcode_and_sdks.sh

```diff
@@ -90,12 +90,7 @@ function package_xcode() {
 
 	pushd ${XCODE}
 
-	EXTRA_ARGS="--exclude=${_name}/usr/bin --exclude=${_name}/Developer/Platforms --exclude=${_name}/usr/lib/sourcekitd.framework --exclude=${_name}/usr/metal"
-	for f in ${_name}/usr/lib/*.dylib
-	do
-		EXTRA_ARGS="--exclude=${f} ${EXTRA_ARGS}"
-	done
-
+	EXTRA_ARGS="--exclude=${_name}/Developer/Platforms --exclude=${_name}/usr/lib/sourcekitd.framework --exclude=${_name}/usr/metal"
```

Comment:
Thank you!

I'll add the second step later, where we'll also output the slimmed down version for linux builds.